extra_data/write_cxi.py

```diff
@@ -38,31 +40,65 @@ def __init__(self, detdata):
             frame_counts.index.values, frame_counts.values.astype(np.intp)
         )
 
-        # cumulative sum gives the end of each train, subtract to get start
+        # Cumulative sum gives the end of each train, subtract to get start
         self.train_id_to_ix = frame_counts.cumsum() - frame_counts
 
     @property
     def nmodules(self):
+        """
+        Number of detector modules.
+        """
         return self.detdata.n_modules
 
+    @property
+    def ncells(self):
```

Comment:
I've got the same naming suggestion as you already implemented for the private attribute (cells_per_entry). But also, I wouldn't bother with making a private attribute and wrapping it in a property - you can just make `self.cells_per_entry` as a normal attribute.

Reply:
Thank you, changed in https://github.com/European-XFEL/EXtra-data/commit/a8c2248ec48f2cbf6a4e7d79afc0b6981c49cb98.